street_network/src/types.rs

```diff
@@ -107,6 +107,14 @@ pub enum InterruptionType {
     DeadEnd,
 }
 
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, PartialOrd, Ord, Serialize, Deserialize)]
+pub enum ConflictType {
```

Comment:
How about a docstring describing what this does? I almost read it as describing how two movements interact or not, but I think it's used to talk about the intersection as a whole

Reply:
Yep :P 

It does describe how two particular movements interact, but `Intersection` `conflict_level` is using that distinction to describe what types of conflict one should expect to find.